packages/@aws-cdk/aws-dynamodb/lib/table.ts

```diff
@@ -712,7 +715,10 @@ abstract class TableBase extends Resource implements ITable {
    * You can customize this by using the `statistic` and `period` properties.
    */
   public metricConsumedWriteCapacityUnits(props?: cloudwatch.MetricOptions): cloudwatch.Metric {
-    return this.metric('ConsumedWriteCapacityUnits', { statistic: 'sum', ...props });
+    return new cloudwatch.Metric({
```

Comment:
I am guessing the next step would be to auto generate the `metricXxx` methods and merge them into the L2 definition (I believe you already managed to do that, right)?

Reply:
Somewhat, and I want to try, but it requires effort on the L2 writer's part so it can't be done automatically.